app/src/main/java/ckmah/mangoreader/UserLibraryHelper.java

```diff
@@ -131,6 +133,17 @@ public void onClick(View v) {
         }
     }
 
+    private static View findMyView(Activity activity) {
```

Comment:
ooo it's what I used to fix the bookmark button, because the associated activity is different.